components/threading/global_threads/global_threads.tsx

```diff
@@ -210,18 +210,7 @@ const GlobalThreads = () => {
                             }, {numUnread})}
                             subtitle={formatMessage({
                                 id: 'globalThreads.threadPane.unreadMessageLink',
-                                defaultMessage: `
-                                    You have
-                                    {numUnread, plural,
-                                        =0 {no unread threads}
-                                        =1 {<link>{numUnread} thread</link>}
-                                        other {<link>{numUnread} threads</link>}
-                                    }
-                                    {numUnread, plural,
-                                        =0 {}
-                                        other {with unread messages}
-                                    }
-                                `,
+                                defaultMessage: 'You have {numUnread, plural, =0 {no unread threads} =1 {<link>{numUnread} thread</link>} other {<link>{numUnread} threads</link>}} {numUnread, plural, =0 {} other {with unread messages}}',
```

Comment:
isn't this change from https://github.com/mattermost/mattermost-webapp/pull/9393 ?

Reply:
Yeah, it is from that branch, I think I have to check how this happened I remember creating branch from master and might have picked it up from stash.

Reply:
Thanks for pointing it out.